examples/fiber/index.html

```diff
@@ -19,26 +19,141 @@
     </div>
     <script src="../../build/react.js"></script>
     <script src="../../build/react-dom-fiber.js"></script>
-    <script>
-      function ExampleApplication(props) {
-        var elapsed = Math.round(props.elapsed  / 100);
-        var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
-        var message =
-          'React has been successfully running for ' + seconds + ' seconds.';
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.24/browser.min.js"></script>
+    <script type="text/babel">
+      var dotStyle = {
+        position: 'absolute',
+        background: '#61dafb',
+        font: 'normal 15px sans-serif',
+        textAlign: 'center',
+        cursor: 'pointer',
+      };
+
+      var containerStyle = {
+        position: 'absolute',
+        transformOrigin: '0 0',
+        left: '50%',
+        top: '50%',
+        width: '10px',
+        height: '10px',
+        background: '#eee',
+      };
+
+      var targetSize = 25;
+
+      class Dot extends React.Component {
+        constructor() {
+          super();
+          this.state = { hover: false };
+        }
+        enter() {
+          this.setState({
+            hover: true
+          });
+        }
+        leave() {
+          this.setState({
+            hover: false
```

Comment:
It causes the props to revert to their initial value when they were first rendered. Something must be screwing up the reuse since the original props shouldn't be around for that long.